source/jormungandr/jormungandr/interfaces/v1/Schedules.py

```diff
@@ -80,13 +80,16 @@ def __init__(self, endpoint):
                                 description="Maximum number of schedule per\
                                 stop_point/route")
         parser_get.add_argument("forbidden_id[]", type=unicode,
-                                description="forbidden ids",
-                                dest="forbidden_uris[]",
-                                action="append")
+                                description="DEPRECATED, replaced by forbidden_uris[]",
+                                dest="forbidden_id[]",
```

Comment:
Minor: I would have name this temporary list with a less usual name (`__temporary_id` ?)  to make sure it is not used in the other part of the code (it's somewhat difficult to know which one to use between forbidden_id and forbidden_uri :wink: )